canonical-websites/insights.ubuntu.com
# Updated look of search results

## Done

- Updated look of search results with the banner like snapcraft.io and the results like ubuntu.com

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [search results](http://0.0.0.0:8023/?search=dell)

## Issue / Card

Fixes #124
